packages/cudos-cmd/cudos-cmd.js

```diff
@@ -21,12 +22,28 @@ async function main() {
     .strict() // checks if the command or optional parameters are specified, if not - user friendly error
     .showHelpOnFail(true) // show help automatically
     .help()
+    .fail((message, error) => {
+      // yargs error message goes here so this is a way to check if error is from yargs
+      if (message) {
+        // Do we want to show help on any error?
+        yargs.showHelp()
+        process.stderr.write(message + '\n')
```

Comment:
is there a good reason to use process.stderr.write here and console.error on line43?

Reply:
no apparent reason, I can change to `console.error `for consistency